docs/user-guide/work-packages/share-work-packages/README.md

```diff
@@ -70,7 +70,11 @@ Once a work package has been shared, you will see the number of shared users in
 
 ## Remove sharing privileges
 
-You can also remove the user from the list by clicking on **Remove** next to the user name. Please note that this will not remove a user entirely, but only revoke the viewing and/or editing rights for the work package. User account will remain intact. If you need to [delete a user](../../../system-admin-guide/users-permissions/users/#delete-users), please do that in system administration or contact your administrator.
+You can also remove the user from the list by clicking on **Remove** next to the user name. Please note that this will not remove a user entirely, but only revoke the viewing and/or editing rights for the work package. User account will remain intact. 
```

Comment:
There is also a "Comment" role so maybe we could change the second sentence to "...only revoke the rights for this work package."